cl/alerts/tasks.py

```diff
@@ -168,25 +195,26 @@ def send_docket_alert(
         delete_redis_semaphore("ALERTS", make_alert_key(d_pk))
         return
 
-    # Notify every user that's subscribed to this alert.
-    messages = make_alert_messages(
-        d, new_des, email_addresses, recap_email_recipients
-    )
+    docket_alert_messages = make_alert_messages(d, new_des, da_recipients)
+    messages = docket_alert_messages
```

Comment:
What's this about?

Reply:
Oh yeah, that's not needed, I've removed it.